functionary/ui/views/function.py

```diff
@@ -102,6 +104,18 @@ def execute(request: HttpRequest) -> HttpResponse:
                     code="invalid",
                 ),
             )
+        except MaxRetryError as err:
```

Comment:
If I interpreted this right, I made the error message that gets rendered to the user be the generic nonsense error, while the error message that gets logged be the more specific error message.

Reply:
Yeah, it was close.  The generic user facing error really only belongs in the view code.  The exception raised inside MinioInterface should be the developer / system administrator relevant one.  I just went ahead and tweaked it.  Otherwise it all looks good.